keyboards/trojan_pinata/model_b/rev0/rev0.c

```diff
@@ -0,0 +1,589 @@
+/* Copyright 2024 Brian H (@Trojan_Pinata)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "rev0.h"
+
+enum oled_modes {
+    MTNS, INFO, LOGO, MEW, CEL, RAY
+};
+
+#define NUM_MODES 6
+uint8_t display_mode = INFO;    // defaults to 1 in case of no value
+
+// key that changes OLED
+#define OLED_MOD KC_F23
```

Comment:
We shouldn't use existing keycodes, as they may be used by the user.  There is a predefined range that can be used, so that should be used.
```suggestion
enum keeb_codes {
    OLED_MOD = QK_KB,
};
```